packages/api/src/beacon/routes/beacon/pool.ts

```diff
@@ -302,6 +400,63 @@ export function getDefinitions(config: ChainForkConfig): RouteDefinitions<Endpoi
       },
       resp: EmptyResponseCodec,
     },
+    submitPoolAttesterSlashingsV2: {
+      url: "/eth/v2/beacon/pool/attester_slashings",
+      method: "POST",
+      req: {
+        writeReqJson: ({attesterSlashing}) => {
+          const fork = config.getForkName(Number(attesterSlashing.attestation1.data.slot));
+          return {
+            body:
+              ForkSeq[fork] >= ForkSeq.electra
+                ? ssz.electra.AttesterSlashing.toJson(attesterSlashing as electra.AttesterSlashing)
+                : ssz.phase0.AttesterSlashing.toJson(attesterSlashing as phase0.AttesterSlashing),
```

Comment:
why the type casts here?

```suggestion
                ? ssz.electra.AttesterSlashing.toJson(attesterSlashing)
                : ssz.phase0.AttesterSlashing.toJson(attesterSlashing),
```